### ISSUE TITLE
Decide on results for Paper 2, assign tasks

### ISSUE THREAD

**Issue by jradavenport** [jradavenport/Gender-in-Astro#14]
### Things I'm interested in putting in Paper 2:
- An overview of the project and it's goals/mission. Definitely need help in deciding what these are, how long project should go for, how it should evolve, what does it need, etc... For example, next time we do the survey, we should make small/cheap flyers or cards to have laying everywhere at the meeting.
- Basic demographics numbers for talks vs speakers vs session chairs vs AAS attendees, as in [Davenport (2014)](http://arxiv.org/abs/1403.3091) (aka Paper 1), and [Pritchard (2014)](http://astrogeo.oxfordjournals.org/content/55/6/6.8).
- some comments on AAS vs NAM and other conferences
- is there "threshold"? If the first Q is F, subsequent Q's are more likely to be F?
- new grid plots?
- Any changes over 3 years of bulk demographic info?
- Group sessions by broad subject, any interesting big statements we can make (w/o being overly incendiary)?
- any actionable ideas we can draw from this data? Trends we can say point to "better" Q/A sessions?
### Big-philosophical-question:

Are we reaching the limit of what we can actually accomplish at AAS with this grass-roots work? The project is at an interesting impasse... do we need a professional (ethnographer/sociologist/etc) to better guide this?

**Comment by malpaslan** [comment]
I think there definitely is some merit in drawing recommendations from the data for future meetings. For example, given that we know that women are more likely to ask questions if the first question is from a woman, and that chairs often find an opportunity to ask the first question… it would follow that more sessions should be chaired by women.

**Comment by jradavenport** [comment]
Yeah, these are exactly the kinds of observations I would like to put in to writing!

```
"We find that sessions chaired by women have more women asking questions."
```

and then in the conclusions

```
"We recommend trying to recruit as many women for session chairs as possible to help equilibrate the Q/A"
```

**Comment by malpaslan** [comment]
I think we definitely have the data to back that statement up (especially in the partial mosaic plots that show just question vs 1st question). It's unclear to me how we could precisely quantify that, but I'd be happy to spend some time digging into the mosaic plot code in R to see if it is possible.

**Comment by jradavenport** [comment]
My organizational thought is: People shout out what results they want to see computed (and if they willing to help do them) and we'll make an "Issue" for each.

**I also want to include ALL previous contributors on the authors list!**
cc: everyone who's worked on the Gender project before, including...

@mehmsy @willettk @karraki @ermaier @hmchristenson @jneilcottle
@astronomeralex @kpoppenhaeger @eringrand 
@mfouesneau @aburgasser @lauralwatkins @PaulDeStefano @bmorris3 @brittafiore @aholachek

Please let me know if I'm missing somebody!

**Comment by malpaslan** [comment]
Cool. Shall I start an 'issue' for the mosaic plots then?

**Comment by jradavenport** [comment]
Yes! :+1:

**Comment by bmorris3** [comment]
I'm willing to contribute whatever I can help with. I'm hoping some ideas come up below that I can take a swing at :metal:

**Comment by mfouesneau** [comment]
Happy to help as well. Let me know what/how to proceed.

fyi remember to check for agreements on publishing the paper. May takes time.

**Comment by jradavenport** [comment]
Good point @mfouesneau, will only include authors who reply with positive consent!

**Comment by astronomeralex** [comment]
happy to be a part of this, too! what can i do to help?

**Comment by eringrand** [comment]
Happy to join up again! (Especailly now I'm much more profiecent in Python and R.)

**Comment by lauralwatkins** [comment]
I'm also very happy to help!

**Comment by malpaslan** [comment]
So as a total aside, some friends of mine are developing a web-based threaded chat platform that works like a mix of chat, forums, and mailing lists. If people are interested, I can get us a room in there to make informal discussions a little easier. It runs in browser, has image embedding, and messages persist between sessions, and we can have different threads running simultaneously in real time. It's on euphoria.io - take a look and let me know, and I'll ask for a room if there is interest.

**Comment by brittafiore** [comment]
Very excited to see this project moving forward. I'm happy to help. Let me
know what I can do!

Britta

On Mon, Jan 25, 2016 at 1:41 PM, Mehmet notifications@github.com wrote:

> So as a total aside, some friends of mine are developing a web-based
> threaded chat platform that works like a mix of chat, forums, and mailing
> lists. If people are interested, I can get us a room in there to make
> informal discussions a little easier. It runs in browser, has image
> embedding, and messages persist between sessions, and we can have different
> threads running simultaneously in real time. It's on euphoria.io - take a
> look and let me know, and I'll ask for a room if there is interest.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jradavenport/Gender-in-Astro/issues/14#issuecomment-174677638
> .

## 

Brittany Fiore-Gartland, Ph.D.
Data Science Ethnographer
Moore-Sloan Data Science Postdoctoral Fellow
Washington Research Foundation Innovation Postdoctoral Fellow
Human Centered Design & Engineering | eScience Institute
University of Washington
@brittafiore

**Comment by PaulDeStefano** [comment]
Thanks for including me!  I know I'm a little removed, now, but I'm willing help any way I can.  I wrote the names-to-gender code for AAS225; if the format of attendee names data for recent conferences is the same, I think it will work.  Do you still need something like that?

**Comment by kpoppenhaeger** [comment]
Hey, great to see that there's a new project! I'm happy to help. I wasn't
at this year's AAS winter meeting because I just moved to the UK - is the
new dataset similar to the one used in our old paper?

Cheers,

Katja

On Mon, Jan 25, 2016 at 10:04 PM, Paul DeStefano notifications@github.com
wrote:

> Thanks for including me! I know I'm a little removed, now, but I'm willing
> help any way I can. I wrote the names-to-gender code for AAS225; if the
> format of attendee names data for recent conferences is the same, I think
> it will work. Do you still need something like that?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jradavenport/Gender-in-Astro/issues/14#issuecomment-174688341
> .

## 

Dr. Katja Poppenhaeger

Lecturer (Assistant Professor)
Queen's University Belfast
Astrophysics Research Centre
Belfast, BT7 1NN, United Kingdom

Research Associate
Harvard-Smithsonian Center for Astrophysics
60 Garden Street
Cambridge, 02138 MA, USA

**Comment by jradavenport** [comment]
@kpoppenhaeger Yup, data should be very similar to what we used a couple years ago. Now that you're in the UK, we hope you'll help carry the project on to NAM :)

To all the project alumni: please do share your thoughts, make Issues, etc! Loving the feedback all

**Comment by jradavenport** [comment]
I've put up a brain dump of avenues that need work. Please, read thru, if you see any you can help attack then make a comment or shoot me a msg.

**Comment by aburgasser** [comment]
I can try to reach out to some of my sociology/ethnic studies/gender
studies colleagues here, to see if they have some insight on how we might
proceed from a broader theoretical basis.  Thanks for resurrecting my
participation!

adam

On Tue, Jan 26, 2016 at 3:45 PM, James Davenport notifications@github.com
wrote:

> I've put up a brain dump of avenues that need work. Please, read thru, if
> you see any you can help attack then make a comment or shoot me a msg.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jradavenport/Gender-in-Astro/issues/14#issuecomment-175295549
> .

## 

---

Adam Burgasser, PhD

Professor of Physics
Center for Astrophysics and Space Sciences
University of California, San Diego

http://cass.ucsd.edu/~ajb

**Comment by ermaier** [comment]
I’m definitely still interested in helping with this! Been super caught up in internship applications, but I should be more free soon, so please let me know what I can do to help.

Erin

**Comment by jradavenport** [comment]
Hey folks, just wanted to ping to ask if there had been any progress? I've talked with a few of you, but want to make sure things are moving forward from people who want to help out.

If things stall out (no judgement - we all over subscribed) I'll push forward with a slightly limited scope. 

Cheers!

**Comment by kpoppenhaeger** [comment]
Hi Jim,

I couldn't push this directly to the repo, but here is the raw data and
participant list I got from the UK exoplanet community meeting 2016v
(UKEXOM2016).

Cheers,

Katja

On Wed, Mar 9, 2016 at 1:45 AM, James Davenport notifications@github.com
wrote:

> Hey folks, just wanted to ping to ask if there had been any progress? I've
> talked with a few of you, but want to make sure things are moving forward
> from people who want to help out.
> 
> If things stall out (no judgement - we all over subscribed) I'll push
> forward with a slightly limited scope.
> 
> Cheers!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jradavenport/Gender-in-Astro/issues/14#issuecomment-194060249
> .

## 

Dr. Katja Poppenhaeger

Lecturer (Assistant Professor)
Queen's University Belfast
Astrophysics Research Centre
Belfast, BT7 1NN, United Kingdom

Research Associate
Harvard-Smithsonian Center for Astrophysics
60 Garden Street
Cambridge, 02138 MA, USA

talk_no,speaker,chair,questions
1,m,f,ffmm
2,m,f,mfff
3,m,f,mmfmm
4,m,m,mmfm
5,f,m,mfm
6,f,m,mm
7,f,m,ffm
8,m,m,m
9,f,f,fmfmm
10,m,f,mf
11,m,f,mfff
12,m,f,mmm
13,f,m,mmfm
14,m,m,mmf
15,m,m,mmf
16,m,m,fmf
17,m,m,m
18,f,m,mmmmm
19,m,m,ffm
20,m,m,mm
21,m,f,mf
22,f,f,mfmf
23,m,f,f
24,m,f,mmf
25,m,m,fm
26,m,m,fff
27,f,m,mf
28,m,m,m
29,m,m,mmm
30,m,m,mmmm
31,m,m,mmm
32,m,m,mm
33,m,m,mm
34,m,m,mf
35,m,m,mm
36,m,f,mmfmmmf
37,m,f,f
38,m,f,mm
39,f,m,mm
40,m,m,m

Conny Aerts
Suzanne Aigrain
David Anderson
Isabelle Baraffe
Adrian Barker
Joanna Barstow
Jayne Birkby
Mariangela Bonavita
Amy Bonsor
Rachel Booth
Ian Boutle
David Brown
Matthew Browning
Andrew Cameron
Tiago Campante
Philip Carter
Sarah Casewell
Heather Cegla
Gilles Chabrier
William Chaplin
Gavin Coleman
Joseph Cooper
Guy Davies
Ernst de Mooij
Brice-Olivier Demory
Martin Dominik
Benjamin Drummond
Cornelis Dullemond
Stephen Durkan
Daniel Evans
Thomas Evans
Mark Fletcher
Clemence Fontanive
Duncan Forgan
Emma Foxell
Ryan Garland
Siddharth Gandhi
Neale Gibson
Samuel Gill
Edward Gillen
Jayesh Goyal
Maximilian Günther
Cassandra Hall
Richard Hall
Paul Hallam
Kirstin Hay 
Raphaëlle Haywood
Éric Hébrard
Christina Hedges
Coel Hellier
Christiane Helling
Sasha Hinkley
Gabriella Hodosán
Mark Hollands
Matthew Hooton
Tiffany Kataria
Grant Kennedy
George King
James Kirk 
Jessica Kirkby-Kent
Stefan Kraus
Guillaume Laibe
Kristine Lam
Graham Lee
Stefan Lines
Joe Llama
Emma Longstaff
Eric Lopez
Pablo Loren-Aguilar
Thomas Louden
Mia Mace
Nikku Madhusudhan
James Manners
Mark Marley
Soko Matsumura
Elisabeth Matthews
Pierre Maxted
Nathan Mayne
Stephane Mazevet
James McCormac
Will McLean
Andrea Miglio
Teo Mocnik
Subhanjoy Mohanty
Samuel Morrell
Annelies Mortier
David Mouillet
Alexander Mustill
Matthew Mutter
Tim Naylor
Richard Nelson
Marion Neveu Van Malle
Charlotte Norris
Hugh Osborn
Sijme-Jan Paardekooper
Paul Palmer
Ian Parry
Hannu Parviainen
Enzo Pascale
Arazi Pinhas
Eva Plávalová
Don Pollacco
Benjamin Pope
Katja Poppenhaeger
Didier Queloz
Vinesh Rajpul
Marco Rocchetto
Pamela Rowden
Sarah Rugheimer
Felix Sainsbury-Martinez
Mark Salisbury
Subhajit Sarkar
Damien Ségransan
Andrew Shannon
David Sing
David Skålid Amundsen
Barry Smalley
Ignas Snellen
John Southworth
Jessica Spake
Daniel Staab
John Strachan
Lorna Temple
Marcell Tessenyi
Jean Teyssandier
Scott Thomas
Andrew Thompson
Samantha Thompson
Giovanna Tinetti
Pascal Tremblin
Amaury Triaud
Angelos Tsiaras
Oliver Turner
Geoffrey Vallis
Dimitri Veras
Colin Vincent
Johanna Vos
Hannah Wakeford
Ingo Waldmann
Christopher Watson
Richard West
Peter Wheatley
Tom Wilson
Jack Yates
John Young
Sergey Yurchenko
University College London
s.yurchenko@ucl.ac.uk

**Comment by jradavenport** [comment]
Hi Katja,

It doesn't look like any data got attached when you replied (I think GitHub strips out any attachments when replying to a comment thread). 

Could you email the data to me directly in a new email?

Thanks!
-Jim
